frontend/src/workbench/Site.tsx

```diff
@@ -11,128 +11,119 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
-import MuiAccordion from '@material-ui/core/Accordion';
-import AccordionDetails from '@material-ui/core/AccordionDetails';
-import MuiAccordionSummary from '@material-ui/core/AccordionSummary';
-import MuiCollapse, { CollapseProps } from '@material-ui/core/Collapse';
-import { makeStyles, withStyles } from '@material-ui/core/styles';
-import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import IconButton from '@material-ui/core/IconButton';
+import { makeStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+import Typography from '@material-ui/core/Typography';
 import React, { useState } from 'react';
 import { SiteProps } from './Site.types';
 
 const useSiteStyles = makeStyles((theme) => ({
-  site: {
-    display: 'flex',
-    flexDirection: 'column',
-  },
-  expanded: {
-    flexGrow: 1,
-    overflow: 'auto',
-  },
-  accordionDetailsRoot: {
-    display: 'block',
-    padding: '0px',
+  leftSite: {
+    display: 'grid',
+    gridTemplateColumns: '32px 1fr',
+    justifyItems: 'stretch',
+    alignItems: 'stretch',
   },
-}));
-
-const Accordion = withStyles({
-  root: {
+  rightSite: {
     display: 'grid',
-    gridTemplateColumns: 'minmax(0,1fr)',
-    gridTemplateRows: 'min-content minmax(0,1fr)',
-    border: '1px solid rgba(0, 0, 0, .125)',
-    boxShadow: 'none',
-    '&:not(:last-child)': {
-      borderBottom: '0px',
-    },
-    '&:before': {
-      display: 'none',
-    },
-    '&$expanded': {
-      margin: '0px',
-    },
+    gridTemplateColumns: '1fr 32px',
+    justifyItems: 'stretch',
+    alignItems: 'stretch',
   },
-  expanded: {},
-})(MuiAccordion);
-
-const AccordionSummary = withStyles({
-  root: {
-    borderBottom: '1px solid rgba(0, 0, 0, .125)',
-    marginBottom: -1,
-    '&$expanded': {
-      minHeight: 48,
-    },
+  viewSelector: {
+    display: 'flex',
+    flexDirection: 'column',
+    background: 'var(--blue-lagoon-lighten-60)',
```

Comment:
Please use a color from the theme (with `theme.palette.xxx.yyy`), we have almost removed all references to the old color system to leverage the MaterialUI one. You may find yourself in a situation where a new color is required (I really don't know).